modules/gitcoinPassportEligibility.json

```diff
@@ -0,0 +1,252 @@
+{
+  "name": "Gitcoin Passport Eligibility",
+  "details": [
+    "A Hats Protocol eligibility module that checks if an address has a minimum score that passes the GitcoinPassportDecoder or custom threshold."
+  ],
+  "links": [
+    {
+      "label": "GitHub",
+      "link": "https://github.com/daocoa/gitcoin-passport-eligibility/"
+    }
+  ],
+  "parameters": [
+    {
+      "label": "Gitcoin Passport Decoder",
+      "functionName": "gitcoinPassportDecoder",
+      "displayType": "default",
+      "REQUESTING_INSIGHT^^^^^^^": "This is represented as an address in the module smart contract. Is this correct or should I use `token`?"
```

Comment:
I think `default` is good here. The `token` display type was used by the Hats app in order to display ERC20 token addresses. But we actually changed that display type to `erc20` and forgot to update the readme, thanks!